workers/ohsome_quality_analyst/api/api.py

```diff
@@ -228,12 +228,18 @@ async def _fetch_report(
 
 
 @app.get("/regions")
-async def get_available_regions():
+async def get_available_regions(geojson=False):
     """List names of available regions."""
     response = empty_api_response()
```

Comment:
response should be the template and should be updated by either with the regions as GeoJSON dict or otherwise write the list of names and features IDs to the `result` field. Currently you either return the response dict (updated with GeoJSON) or just a `list`. 

```
if geojson:
    response.update(regions)
else: 
    response["result"] = regions_name_list
```